tests/test_cli.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import sys
+from unittest.mock import patch
+from ldfparser.cli import main
+
+
+@pytest.mark.parametrize('command', [
```

Comment:
Should be tested for different LDFs as well, maybe there's some edge case where a field is missing.